addons/web/controllers/pivot.py

```diff
@@ -102,9 +102,12 @@ def export_xlsx(self, data, token):
 
         workbook.close()
         xlsx_data = output.getvalue()
+        filename = _("Pivot of %(screen_title)s (%(model)s).xlsx",
```

Comment:
Spaces are often problematic / inconvenient in filenames (e.g. shell). Maybe use not-spaces there, or replace spaces by underscores or dashes afterwards (to ensure we always generate names without spaces).

Reply:
And the extension *probably* should not be part of the translatable content.

Reply:
> Spaces are often problematic / inconvenient in filenames (e.g. shell). Maybe use not-spaces there, or replace spaces by underscores or dashes afterwards (to ensure we always generate names without spaces).

I think modern operating systems and tools deal correctly with spaces in filenames, and the spaces add to clarity for the average non-technical user. But I don't have a strong opinion either way. That should probably be up to the product owner :)


> And the extension _probably_ should not be part of the translatable content.

I [suggested](https://github.com/odoo/odoo/pull/64123#discussion_r552479682) that because it provides /some/ context to the translator ("this string represents a filename"), and because I expected most translators to know what ".xlsx" is. On second thought this is probably not true, and the added context is not extremely useful, so +1 for taking the ".xslx" outside of the `_()`.

Reply:
> I think modern operating systems and tools deal correctly with spaces in filenames

Higher-level tools deal fine, I'm saying there are situations where it can be inconvenient e.g. most shells will need quoting or escaping somehow. Since we can avoid it, I'd rather we did.

Reply:
How about model names with `/` `\` ? For instance, the description of `hr.leave.report` is `'Time Off Summary / Report'`

Reply:
> How about model names with `/` `\` ? For instance, the description of `hr.leave.report` is `'Time Off Summary / Report'`

@mart-e , we have checked forward slash in linux and mac OS it will automatically replace them with _ or space. but you guys know better so if there are any changes let us know.  Thanks